src/Services/MailLoader/MjmlLoader.php

```diff
@@ -52,22 +52,17 @@ public function load(string $path): string
         } catch (ServerException $e) {
             $this->logger->critical('MJML Api is not accessible', ['response' => $e->getResponse()->getBody(), 'code' => $e->getResponse()->getStatusCode()]);
 
-            throw $e;
+            // Return empty string to load shopware default templates.
+            return '';
         }
 
         $compileTemplate = json_decode($response->getBody()->getContents(), true);
 
-        if (is_null($compileTemplate) || count($compileTemplate) === 0) {
-            // ToDo: Get default mail template contents.
+        if (is_null($compileTemplate) || count($compileTemplate) === 0 || !empty($compileTemplate['errors'])) {
```

Comment:
If the mjml template has error, we should throw an exception. The above catch is for service not available

Reply:
We think that it shouldn't throw an exception, as then there will be no mail sent at all.
As a backup it would be nice if the shop would at least send the default template.
Shouldn't we just log the error and send the sw default one?

Reply:
Ok, we changed that part of the code and enhanced the existing code, as it wouldn't have logged any other errors than the first one.